autopush/tests/test_bridge.py

```diff
@@ -1,7 +1,4 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
+# -*- coding: utf-8 -*-
```

Comment:
Why does no other file in the project have it? The LICENSE file already has the MPL, why does it need to start appearing now? (Oh hey, now that I look, I see we don't have a LICENSE file, one sec, adding it to this PR)